releasenotes/notes/remove-ibmqdeprecatedbackendservice-668f549558e21088.yaml

```
@@ -0,0 +1,6 @@
+---
+upgrade:
+  - |
+    The deprecated :class:`qiskit.providers.ibmq.IBMQDeprecatedBackendService` 
```

```suggestion
    The deprecated `qiskit.providers.ibmq.IBMQDeprecatedBackendService` 
```
The doc link will fail since we are removing the class.